config/intellij-idea-inspections.xml

```diff
@@ -4638,7 +4638,8 @@
     ,com.puppycrawl.tools.checkstyle.internal.TestUtil,assert.*, ,
     ,nl.jqno.equalsverifier.EqualsVerifier,verify.*, ,
     ,org.checkstyle.base.AbstractItModuleTestSupport,verify.*, ,
-    ,com.tngtech.archunit.lang.ArchRule,check"/>
+    ,com.tngtech.archunit.lang.ArchRule,check, ,
+    ,com.puppycrawl.tools.checkstyle.filters.SuppressionFilterTest,testUseCache.*"/>
```

Comment:
should be:
`com.puppycrawl.tools.checkstyle.AbstractModuleTestSupport, execute`

as you show Inspection method that should be considered as "assert"

Reply:
Done